phpcs.xml.dist

```diff
@@ -5,12 +5,318 @@
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityLabel.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityRendered.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityType.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityLanguage.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityUrl.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityLoadMultiple.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityOwner.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityCreated.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityDescription.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityTranslation.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityLoad.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityTranslations.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/Fields/Image/ImageDerivative.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/Fields/Image/ImageUrl.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityChanged.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityId.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityUuid.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityAccess.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityLoadByUuid.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/Entity/EntityBundle.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/GraphQL/Schema/SdlSchemaPluginBase.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/DataProducerPluginManager.php</exclude-pattern>
+    <exclude-pattern>src/Plugin/DataProducerPluginCachingInterface.php</exclude-pattern>
+    <exclude-pattern>src/Controller/RequestController.php</exclude-pattern>
+    <exclude-pattern>examples/graphql_example/src/Wrappers/QueryConnection.php</exclude-pattern>
+    <exclude-pattern>examples/graphql_example/src/Plugin/GraphQL/DataProducer/QueryArticles.php</exclude-pattern>
+    <exclude-pattern>tests/src/Traits/DataProducerExecutionTrait.php</exclude-pattern>
+    <exclude-pattern>tests/src/Kernel/ResolverBuilderTest.php</exclude-pattern>
+    <exclude-pattern>tests/src/Kernel/DataProducer/XML/XMLTestBase.php</exclude-pattern>
+    <exclude-pattern>tests/src/Kernel/DataProducer/SeekTest.php</exclude-pattern>
+    <exclude-pattern>tests/src/Kernel/GraphQLTestBase.php</exclude-pattern>
+  </rule>
+
+  <exclude-pattern>graphql.info.yml</exclude-pattern>
```

Comment:
why do we exclude those yml files? Do we get any errors for them?

Reply:
If I remove <rule ref="Drupal"> phpcs is mad and gives out this error:

No PHP code was found in this file and short open tags are not allowed by this install of PHP. This file may be using short open tags but PHP does not allow them.

Reply:
Ah I see, because phpcs somehow throws errors for them. That happens because you did not refernece a standard, so once you bring back the Drupal reference we should not see this.

Reply:
Jup build works now